core/src/main/resources/hudson/PluginManager/PluginDeprecationMonitor/message.properties

```diff
@@ -0,0 +1,27 @@
+# The MIT License
+#
+# Copyright (c) 2019, Joseph Petersen
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+
+DeprecatedPlugins=\
+  The following plugins have been marked as deprecated.<br />\
```

Comment:
This looks a bit strange in the UI,
It seems like the list is going to immediately follow this line, but instead two other lines follow,

possibly,
```suggestion
  There is deprecated plugins installed.<br />\
```

Reply:
We could move the explain and just have a simpler line for the `dt` element.

Reply:
👍

Reply:
Yup, this look much better:
![image](https://user-images.githubusercontent.com/1661688/74870679-6bdc8e80-535a-11ea-901e-65f561c2387c.png)